templates/cassiopeia/scss/blocks/_form.scss

```diff
@@ -108,7 +108,23 @@ td .form-control {
   bottom: 100%;
 }
 
+fieldset + fieldset {
```

Comment:
can I suggest to write this as follow:

```
fieldset {
  + fieldset {
    margin-top: $cassiopeia-grid-gutter*2;
  }

  > * + * {
    margin-top: $cassiopeia-grid-gutter*2;
  }
}
```

and remove the margin-top on the control-group

Reply:
If this solve the problem of spacing between fieldsets on the forms, we can take it.
You can have a look at "site settings" and "user profile" when you login on the frontend.

Reply:
discussed on Glip.